### ISSUE TITLE
Dropdown high contrast issues

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#3488]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Firexox 56.0 on Windows 10

#### Priorities and help requested (not applicable if asking question):
Are you willing to submit a PR to fix? No
Requested priority: Normal

#### Describe the issue:
When on Windows 10 high contrast mode using the FireFox browser, the dropdown focus is not visible at all. 

User impact: People having low vision will not be able to see the active focus on the combo box.

#### Actual behavior:

Screen shot taken from https://developer.microsoft.com/en-us/fabric#/components/dropdown 
on Firefox browser on Windows 10 high contrast mode.

![image](https://user-images.githubusercontent.com/24945574/33450064-811bc66e-d5bf-11e7-9279-3cb392268304.png)

#### Expected behavior:

The component should have a similar focus indicator on Firefox as in e.g. Edge:

![image](https://user-images.githubusercontent.com/24945574/33450325-4c45fd28-d5c0-11e7-86e3-adee1264de18.png)

**Comment by satbai** [comment]
I think this might be related. Dropdown component on WHITE high contrast mode doesn't show the dropdown icon on Edge. Reproduces here: https://developer.microsoft.com/en-us/fabric#/components/dropdown

![image](https://user-images.githubusercontent.com/24945574/37373754-30d89846-26d5-11e8-8bc2-aee70fae3059.png)

**Comment by aditima** [comment]
@satbai - we really only support the high-contrast mode on IE/Edge. Closing this issue.
